i18n/README.it-it.md

```diff
@@ -63,94 +66,86 @@ Gli utenti macOS possono usare Homebrew:
-  `9000`
-- [REST API](https://questdb.io/docs/reference/api/rest/) sulla porta `9000`
-- [PostgreSQL](https://questdb.io/docs/reference/api/postgres/) wire protocol sulla
-  porta `8812`
-- [InfluxDB](https://questdb.io/docs/reference/api/ilp/overview/) line protocol per
-l'ingestione ad alto rendimento sulla porta `9009`
+- La[Console Web](https://questdb.io/docs/develop/web-console/) che fornisce un editor SQL interattivo sulla porta `9000`
+- Il [protocollo di linea InfluxDB](https://questdb.io/docs/reference/api/influxdb/) per l'ingestione ad alto throughput sulla porta `9009`
+- L’[API REST](https://questdb.io/docs/reference/api/rest/) sulla porta `9000`
+- Il [protocollo di rete PostgreSQL](https://questdb.io/docs/reference/api/postgres/) sulla porta `8812`
 
-## Come è QuestDB in confronto agli altri TSDBs open source
+### Inserisci i tuoi dati
+Di seguito riportiamo i client ufficiali di QuestDB per alcuni dei linguaggi di programmazione più popolari:
 
-Ecco le high-cardinality
-[Time Series Benchmark Suite](https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/)
-risultati ottenuti usando `cpu-only` con 6 workers su un AMD Ryzen 3970X:
+- [.NET](https://github.com/questdb/net-questdb-client)
+- [C/C++](https://github.com/questdb/c-questdb-client)
+- [Go](https://pkg.go.dev/github.com/questdb/go-questdb-client)
+- [Java](https://questdb.io/docs/reference/clients/java_ilp/)
+- [NodeJS](https://questdb.github.io/nodejs-questdb-client)
+- [Python](https://py-questdb-client.readthedocs.io/en/latest/)
+- [Rust](https://docs.rs/crate/questdb-rs/latest)
+
+### Avvio rapido end-to-end
+Se desideri eseguire tutto quello di cui hai bisogno, dall'ingestione in streaming dei dati alla loro visualizzazione con Grafana, dai un'occhiata al nostro [repository di avvio rapido.](https://github.com/questdb/questdb-quickstart).
+
+## QuestDB a confronto con altri database open-source per serie temporali 
+
+[Questo articolo](https://questdb.io/blog/2021/07/05/comparing-questdb-timescaledb-influxdb/) mette a confronto QuestDB con altri database di serie temporali open source in termini di funzionalità, maturità e prestazioni.
+
+Ecco i risultati dell'High-Cardinality Time Series Benchmark Suite che utilizza il caso d'uso "solo CPU" con 6-16 lavoratori su 32 CPU e 64 GB di RAM::
 
 <div align="center">
   <a href="https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/">
-    <img alt="Grafico che compara la massima portata di QuestDB, ClickHouse, TimescaleDB and InfluxDB." width="600" alt="tsbs-results" src="https://user-images.githubusercontent.com/91843271/197382161-e5f3f5b0-18bf-439a-94e4-83ab4bf91d7c.png">
+    <img alt="Grafico che compara la produttività di QuestDB, ClickHouse, TimescaleDB, ed InfluxDB." width="600" alt="tsbs-results" src="https://user-images.githubusercontent.com/91843271/197382161-e5f3f5b0-18bf-439a-94e4-83ab4bf91d7c.png">
```

Comment:
I think that chart is outdated and we are now using this one https://raw.githubusercontent.com/questdb/questdb/master/.github/readme-benchmark.png